acquire/utils.py

```diff
@@ -381,7 +384,12 @@ def normalize_path(target: Target, path: Path, resolve: bool = False, lower_case
         path = path.lower()
 
     if target.os == "windows":
-        # As dissect.target always maps c: onto sysvol, we can do this substitution here.
-        path = SYSVOL_SUBST.sub("sysvol", path)
+        path = DEVICE_SUBST.sub("", path)
+        if sysvol:
+            path = normalize_sysvol(path, sysvol)
```

Comment:
I think it is better to put this before the if condition containing `path.lower()` so the drive letter always has a uniform case.